make-release.sh

```diff
@@ -116,34 +117,35 @@ gitCommit() {
 }
 
 gitPush() {
+  local TARGET_BRANCH=$1
   if  [[ ${DOPUSH} -eq 1 ]]; then 
     git pull origin "${TARGET_BRANCH}" || true
     git push origin "${TARGET_BRANCH}"
   fi
 }
 
 gitPullRequest() {
+  local HEAD_BRANCH=$1
+  local BASE_BRANCH=$2
   if  [[ ${DOCOMMIT} -eq 1 ]]; then 
-    git pull origin "${TARGET_BRANCH}" || true
-    git push origin "${TARGET_BRANCH}"
+    git pull origin "${HEAD_BRANCH}" || true
+    git push origin "${BASE_BRANCH}"
     LASTCOMMITCOMMENT="$(git log -1 --pretty=%B)"
-    hub pull-request --force --message "${LASTCOMMITCOMMENT}" --base "${MAIN_BRANCH}" --head "${TARGET_BRANCH}"
```

Comment:
Are you sure we can omit to specify `--base "${MAIN_BRANCH}"`? As the `main` branch is not the default branch (yet), I would expect the pull request to target the default branch, `master`, which ins not desirable.

Reply:
nice catch!